JEP.md

```diff
@@ -78,11 +78,11 @@ Example event schema
 
 Schema validation is done in the core telemetry framework that routes events from publishers to sinks, so that each event sink does not need to implement schema validation.
 
-**Open Questions/TBD**:
-
-* List of rules for event schemas
-* How do organizations deploy custom schemas?
-* Where are the schemas for public events stored?
+Schemas are used at two places
+* During the emit run-time, for validating events before publishing to the configured event sink.
+  * The schemas are picked up from a side-car package for runtime validation to avoid the network call.
```

Comment:
They could be in the same package, rather than a sidecar.

Reply:
How would this work with custom schemas? I'd envision organizations adding those to  a package that is deployed along side the server and is on the $PATH when we come to resolve the schemas

Reply:
Oh sure. They could be in a sidecar package *or* in the same package. Right now it looks like you need a custom sidecar package (like jupyterhub-events)